src/components/Shortcuts/__snapshots__/test.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`<Shortcuts /> renders without crashing 1`] = `
 >
   <li>
     <ul
-      className="menulist"
+      className="menulist -is-logged-in"
```

Comment:
@ceanela same question as above, this seems to add the class to the ul itself, that would be the preference. We would not need the class added to the li's, just the parent ul.

Reply:
Ignore any code in the snapshots please

Reply:
Will do.